doc/keycmds.dsv

```diff
@@ -13,7 +13,7 @@ prev-unread||p||Jump to the previous unread article.
 next||J||Jump to next list entry.
 prev||K||Jump to previous list entry.
 random-unread||^K||Jump to a random unread article.
-open-in-browser||o||Open the URL associated with the current article, or selection when in the URL view.
+open-in-browser||o||Open the URL associated with the current feed, article, or selection in the browser.
```

Comment:
I think this change lost a bit of information: "selection when in the URL view" turned into "selection", which is very generic. Can you please bring back the mention of the URL view?

Reply:
Well, that was my intention.

I might be stretching Newsboat vocabulary, but to me the term "selection" refers to the highlighted feed in the feed list, the highlighted article in the article list, the highlighted URL in the URL view, etc.

Have a look at the next commit.

Reply:
> to me the term "selection" refers to the highlighted feed in the feed list, the highlighted article in the article list, the highlighted URL in the URL view, etc.

That makes sense, but why keep the explicit mentions of "feed" and "article" then?

Frankly I think we should be going in an opposite direction, replacing "selection" with something more approachable like "the URL selected in the URL view". That's wordier, but also *way* more specific and thus, hopefully, easier to understand.

> Have a look at the next commit.

Um, there's only one commit in this PR?

Reply:
> That makes sense, but why keep the explicit mentions of "feed" and "article" then?

I still wonder even after looking at c2fa0d5.

Reply:
It's uploaded now! Got confused with branches and a detached HEAD...

Reply:
> why keep the explicit mentions of "feed" and "article" then?

When I take a look at the XML file of an RSS feed, I see a `<link>` tag for the feed, and another `<link>` tag for every article. Those links look different then the `<a href=` links present in the article `<content>`, which is mostly what populates the URL view (I think).

Since `open-in-browser` seems to operates transparently for all those, It seemed appropriate to mention feed and article separately.

Please correct me if I'm wrong.

Reply:
I don't think you're wrong, it's just that:

1. you seem to hold two contradictory beliefs at once;
2. I disagree with your proposed wording (even though I see that it does make sense from a certain point of view).

So, first point: these two sentences sound contradictory to me:

> to me the term "selection" refers to the highlighted feed in the feed list, the highlighted article in the article list, the highlighted URL in the URL view, etc.
>
> Since open-in-browser seems to operates transparently for all those, It seemed appropriate to mention feed and article separately.

As far as I understand, the first says that "selection" covers all dialogs (including the feeds and articles), but the second draws a difference between `<link>` and `<a href>` (i.e. between feeds/articles and URLs in articles). Both views are accurate, but I feel that presenting them both makes the doc confusing.

The second point (my disagreement with the wording) comes down to: I don't think it's immediately obvious what "selection" means anymore. Previous wording was more specific, but now a user might wonder if they need to do anything special to "select". That's why I'm pushing for a wording that leaves no room for confusion. c2fa0d5 still falls short on that front, since the phrase "selection applies to many dialogs" naturally makes the user wonder "which ones?" and "many but not all?" and so forth.

Reply:
I agree.

What about:

Open a URL in the browser. Newsboat will choose the URL based on the current _dialog_ (for example, open the URL associated with the selected article while in the _article list_, or open the selected URL while in the _URL view_).

I know we are back to something similar to the initial description. Feel free to close this PR if it's too much info.

Reply:
IMHO this is slightly better than what's in c2fa0d5, but the parenthesised part would still make the user wonder about the behaviour in other dialogs. Since that op only supports four dialogs in total, I don't see why the doc can't just list them all.

> I know we are back to something similar to the initial description.

I don't think this is a bad thing. As I see it, the only two problems with the initial description were: 1) it didn't mention feeds; 2) it didn't make it explicit that the URL is opened in the browser. This PR fixed the first one, but introduced a "regression" of sorts while fixing the second one, and now we're stuck discussing that.

My proposal is this:

> Use browser to open the URL associated with the current article, feed, or entry in the URL view.

How does that sound to you?

Reply:
Sure, sounds good!

> Since that op only supports four dialogs in total, I don't see why the doc can't just list them all.

> 1) it didn't mention feeds;
> 2) it didn't make it explicit that the URL is opened in the browser.

Sums up the PR intention very well.